src/Pages/TheGear/TheGear.js

```diff
@@ -19,13 +19,13 @@ import React from 'react'
 //   CardImg
 // } from 'reactstrap'
 // import classnames from 'classnames'
-import './App.css'
+import './TheGear.css'
```

Comment:
like the line below, use dashes and lowercase
```suggestion
import './the-gear.css'
```